pysat/tests/classes/cls_instrument_library.py

```diff
@@ -480,8 +485,11 @@ def test_load_empty(self, inst_dict):
 
         return
 
+    # TODO(v3.3.0): remove mark.new_tests when ecosystem is fully compliant
+    # for this test
```

Comment:
```suggestion
    # TODO(v3.3.0): remove mark.new_tests
```

Because at this point it would be better for the ecosystem package to add a pysat cap if it wants to keep failing without the mark of failure.